src/main/java/edu/harvard/iq/dataverse/util/SignpostingResources.java

```diff
@@ -0,0 +1,337 @@
+                    da.getIdType(),
+                    da.getIdValue(),
+                    da.getAffiliation(),
+                    da.getIdentifierAsUrl()
+            ));
+
+            String authorURL = "";
+            authorURL = getAuthorUrl(da);
+            if (!Objects.equals(authorURL, "")) {
+                visibleAuthorCounter++;
+                // return empty if number of visible author more than max allowed
+                if (visibleAuthorCounter >= maxAuthors) return "";
+                singleAuthorString = "<" + authorURL + ">;rel=\"author\"";
+                if (Objects.equals(identifierSchema, "")) {
+                    identifierSchema = singleAuthorString;
+                } else {
+                    identifierSchema = String.join(",", identifierSchema, singleAuthorString);
+                }
+            }
+        }
+
+        logger.info(String.format("identifierSchema: %s", identifierSchema));
+        return identifierSchema;
+    }
+
+    /**
+     * Get key, values of signposting items and return as string
+     *
+     * @return comma delimited string
+     */
+    public String getLinks() {
+        List<String> valueList = new LinkedList<>();
+        Dataset ds = workingDatasetVersion.getDataset();
+
+        String identifierSchema = getAuthors(workingDatasetVersion.getDatasetAuthors());
+        if (!identifierSchema.equals("")) {
+            valueList.add(identifierSchema);
+        }
+
+        if (!Objects.equals(ds.getPersistentURL(), "")) {
+            String citeAs = "<" + ds.getPersistentURL() + ">;rel=\"cite-as\"";
+            valueList.add(citeAs);
+        }
+
+        List<FileMetadata> fms = workingDatasetVersion.getFileMetadatas();
+        String items = getItems(fms);
+        if (items != null && !Objects.equals(items, "")) {
+            valueList.add(items);
+        }
+
+        String describedby = "<" + describedByJsonObj.getString(ds.getProtocol()) + ds.getAuthority() + "/"
+                + ds.getIdentifier() + ">;rel=\"describedby\"" + ";type=\"" + describedByJsonObj.getString("type") + "\"";
+        describedby += ",<" + systemConfig.getDataverseSiteUrl() + "/api/datasets/export?exporter=schema.org&persistentId="
+                + ds.getProtocol() + ":" + ds.getAuthority() + "/" + ds.getIdentifier() + ">;rel=\"describedby\"" + ";type=\"application/json+ld\"";
+        valueList.add(describedby);
+
+        String type = "<https://schema.org/AboutPage>;rel=\"type\"";;
+        if (useDefaultFileType) {
+            type = "<https://schema.org/AboutPage>;rel=\"type\",<" + defaultFileTypeValue + ">;rel=\"type\"";
+        }
+        valueList.add(type);
+
+        // TODO: support only CC0 now, should add flexible license support when flex-terms or multi-license is ready
+//        License license = workingDatasetVersion.getTermsOfUseAndAccess().getLicense();
+        License license = workingDatasetVersion.getTermsOfUseAndAccess().getLicense();
```

Comment:
License is null when custom terms are used. The DataUtil method will get you a URI for both standard and custom licenses.